core/templates/pages/exploration-player-page/learner-experience/input-response-pair.component.css

```diff
@@ -22,9 +22,6 @@
   width: 100px;
 }
 
-oppia-input-response-pair .oppia-popover {
```

Comment:
Why did you delete this?

Reply:
@seanlip These Changes were made by me while testing. And I forgot to undo these changes. I will correct it.